core/ui/public/i18n/en/translation.json

```diff
@@ -938,7 +939,10 @@
         "update_to_stable_version": "Update to stable version",
         "update_to_testing_version": "Update to testing version",
         "num_instances_installed": "{num} instance installed | {num} instances installed",
-        "reason_max_per_node_limit": "Limit of {param} instance reached | Limit of {param} instances reached"
+        "reason_max_per_node_limit": "Limit of {param} instance reached | Limit of {param} instances reached",
+        "core_app": "Core apps",
```

Comment:
Minor improvement: if possible, let's keep key and value consistent, e.g. `core_apps` (trailing "s")